src/js/analytics.js

```diff
@@ -7,9 +7,9 @@ function trackLinkClick(element) {
 }
 
 function startAnalytics() {
-  const consentCookie = getConsentCookie();
+  const cookiePreferences = getCookiePreferences();
 
-  if (consentCookie.statistics) {
+  if (cookiePreferences.hasAcceptedOptionalCookies) {
```

Comment:
Silly question but I assume this is the same as whats on spa-login right?

Reply:
Yes. The new cookie contents on spa-login has two values in it: `hasSeenBanner `and `hasAcceptedOptionalCookies`.